extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/PeertubeParsingHelper.java

```diff
@@ -27,6 +28,7 @@ public static Calendar parseDateFrom(String textualUploadDate) throws ParsingExc
         Date date;
         try {
             date = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.S'Z'").parse(textualUploadDate);
+            date = new Date(date.getTime() + TimeUnit.HOURS.toMillis(1));
```

Comment:
This is dependent on timezone of the user.
 Since PeerTube returns GMT  response `sdf.setTimeZone(TimeZone.getTimeZone("GMT"))` should do the trick.

Reply:
Makes sense as I'm GMT+1. Thanks for the help!